### PR TITLE
adding note about contacting us/checking spam

### PR DESCRIPTION
**Pull request by ChristinaLK**

### CODE REVIEW

**Comment by ChristinaLK** [comment]
@lmichael107 Jason + I discussed adding some text to the page that you see after filling out the form, telling people to contact us if they don't hear from us. This is first draft language so please comment away. :)

File: thanks.md [hunk 1]
```diff
@@ -8,5 +8,10 @@ var RecaptchaOptions = {
...
 
-Thank you for your account request! A <a href="get-help.shtml">research 
-computing facilitator</a> will get in touch with you soon.
+Thank you for your account request! You should get an email with "[CHTC Requests]" in the subject line within the next 
+few minutes and a <a href="get-help.shtml">research 
+computing facilitator</a> be in touch within a business day. 
+
+If you don't get an email with the "CHTC Requests" subject or hear from a research computing facilitator within a 
+day, please double check your spam folder to confirm that these emails did not go to spam; if they aren't in spam, 
+please contact us at our main email address at the bottom of this website to confirm that we received your account request.
```
Comment (lmichael107) [inline thanks.md:17]:
```suggestion
please contact us via the email address at the bottom of this page to confirm that we received your account request. 
```

File: thanks.md [hunk 2]
```diff
@@ -8,5 +8,10 @@ var RecaptchaOptions = {
...
 </script>
 
-Thank you for your account request! A <a href="get-help.shtml">research 
-computing facilitator</a> will get in touch with you soon.
+Thank you for your account request! You should get an email with "[CHTC Requests]" in the subject line within the next 
+few minutes and a <a href="get-help.shtml">research 
+computing facilitator</a> be in touch within a business day. 
+
+If you don't get an email with the "CHTC Requests" subject or hear from a research computing facilitator within a 
+day, please double check your spam folder to confirm that these emails did not go to spam; if they aren't in spam,
```
Comment (lmichael107) [inline thanks.md:16]:
```suggestion
day, please check your spam folder; if they aren't in spam, 
```

**Review by lmichael107 — COMMENTED** [review]
A few changes to tighten up wording, but looks good!

File: thanks.md [hunk 3]
```diff
@@ -8,5 +8,10 @@ var RecaptchaOptions = {
 };
 </script>
 
-Thank you for your account request! A <a href="get-help.shtml">research 
-computing facilitator</a> will get in touch with you soon.
+Thank you for your account request! You should get an email with "[CHTC Requests]" in the subject line within the next 
+few minutes and a <a href="get-help.shtml">research 
+computing facilitator</a> be in touch within a business day.
```
Comment (ChristinaLK) [inline thanks.md:13]:
```suggestion
computing facilitator</a> should be in touch within a business day. 
```

**Pull request merged**